src/gluonts/dataset/repository/_util.py

```diff
@@ -59,11 +59,15 @@ def save_to_file(path: Path, data: List[Dict]):
 def metadata(
     cardinality: Union[int, List[int]], freq: str, prediction_length: int
 ):
+    if isinstance(cardinality) != list:
```

Comment:
```suggestion
    if not isinstance(cardinality, list):
```

Reply:
I messed up my environment. It didn't catch that. Thanks.